substrate/frame/multisig/src/weights.rs

```diff
@@ -69,12 +72,12 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	/// The range of component `z` is `[0, 10000]`.
 	fn as_multi_threshold_1(z: u32, ) -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `145`
+		//  Measured:  `0`
 		//  Estimated: `3997`
-		// Minimum execution time: 28_800_000 picoseconds.
```

Comment:
All of the weights have improved (i.e. lowered).

What happened here, does this mean the `multisig` pallet has had incorrect weights up until this PR?

Reply:
not sure tbh :)